src/applications/personalization/view-dependents/manage-dependents/tests/unit/ManageDependentsApp.unit.spec.jsx

```diff
@@ -0,0 +1,138 @@
+      />,
+      {
+        initialState,
+        reducers: { removeDependents },
+      },
+    );
+    expect(await screen.findByText('Remove dependent', { selector: 'button' }))
+      .to.exist;
+    expect(await screen.findByLabelText('Divorce')).to.exist;
+    expect(await screen.findByLabelText('Annulment')).to.exist;
+    expect(await screen.findByLabelText('Declared Void')).to.exist;
+    expect(await screen.findByLabelText('Spouse’s Death')).to.exist;
+    expect(await screen.findByText('Date marriage ended')).to.exist;
+    expect(await screen.findByText('State where this happened')).to.exist;
+    expect(await screen.findByText('City where this happened')).to.exist;
+  });
+
+  it('fills out the form', async () => {
+    const initialState = {
+      removeDependents: {
+        dependentsState: {
+          1: {
+            uiSchema: SCHEMAS.Spouse.uiSchema,
+            formSchema: SCHEMAS.Spouse.schema,
+            formData: {},
+          },
+        },
+      },
+      user: {
+        profile: {
+          vapContactInfo: {
+            email: {
+              emailAddress: 'someone@famous.com',
+            },
+            homePhone: {
+              areaCode: '555',
+              phoneNumber: '8001212',
+            },
+            residentialAddress: {
+              addressType: 'DOMESTIC',
+              countryName: 'United States',
+              countryCodeIso3: 'USA',
+              addressLine1: '123 Main Blvd',
+              addressLine2: 'Floor 33',
+              addressLine3: 'Suite 55',
+              city: 'Hollywood',
+              stateCode: 'CA',
+              zipCode: '90210',
+            },
+          },
+        },
+      },
+    };
+
+    const screen = renderInReduxProvider(
+      <ManageDependentsApp
+        relationship="Spouse"
+        closeFormHandler={() => {}}
+        stateKey={1}
+        userInfo={{
+          fullName: {
+            firstName: mockData.firstName,
+            lastName: mockData.lastName,
+          },
+          dateOfBirth: mockData.dateOfBirth,
+          ssn: mockData.ssn,
+        }}
+      />,
+      {
+        initialState,
+        reducers: { removeDependents },
+      },
+    );
+    userEvent.click(await screen.findByLabelText('Divorce'));
+    userEvent.selectOptions(await screen.findByLabelText('Month'), ['1']);
+    userEvent.selectOptions(await screen.findByLabelText('Day'), ['25']);
+    userEvent.type(await screen.findByLabelText('Year'), '2003');
+    userEvent.selectOptions(
+      await screen.findByLabelText('State where this happened', {
+        exact: false,
+      }),
+      ['AL'],
+    );
+    userEvent.type(
+      await screen.findByLabelText('City where this happened', {
+        exact: false,
+      }),
+      'Test city',
+    );
+    const submitBtn = screen.findByText('Remove dependent', {
+      selector: 'button',
+    });
+    userEvent.click(await submitBtn);
+    expect(await screen.queryAllByRole('alert')).to.have.lengthOf(0);
```

Comment:
is this meant to confirm there are no form validation errors?

also, no need for the `await` in front of `query*`

Reply:
Yeah, meant to confirm no validation errors. Can you think of a better way to do that? I wanted to search by error class name, but didn't see a way to do that with `query*` or `find*`.

Reply:
Nope! I've never done a test like that, as far as I recall. Well, I guess I have checked to make sure that the errors DO show up https://github.com/department-of-veterans-affairs/vets-website/blob/e11b659aed7307fc9abf282bdfc26416e16fc3ad/src/applications/personalization/profile/tests/e2e/personal-and-contact-info/mailing-address.cypress.spec.js#L65